UIKit_DZ/UIKit_DZ/MenuItemConfigScreen/MenuItemCofigurationViewController.swift

```diff
@@ -3,5 +3,176 @@
+            style: .done,
+            target: self,
+            action: nil
+        )
+        navigationItem.rightBarButtonItem = UIBarButtonItem(
+            image: UIImage(systemName: "paperplane.fill"),
+            style: .done,
+            target: self,
+            action: #selector(activityButtonTapped)
+        )
+    }
+
+    /// кладем элементы на экран
+    private func setupUi() {
+        view.backgroundColor = .white
+        topCustomView.addSubview(backButtonBackgroundView)
+        topCustomView.addSubview(itemImage)
+        createLabel(label: modifireLabel, lineX: 16, lineY: 432, title: Constants.modifireLabel)
+        createLabel(label: priceLabel, lineX: 16, lineY: 669, title: "Цѣна - \(price) руб")
+        orderMethod()
+        createSegmendControll()
+        emptyView(title: "Обжарка", lineX: 15)
+        emptyView(title: "Дополнительные /n ингредіенты", lineX: 195)
+    }
+
+    /// создание лейблов
+    private func createLabel(label: UILabel, lineX: Int, lineY: Int, title: String, fontSize: CGFloat = 16) {
+        label.frame = CGRect(x: lineX, y: lineY, width: 345, height: 31)
+        label.text = title
+        label.font = UIFont(name: "HelveticaNeue-Bold", size: fontSize)
+        priceLabel.textAlignment = .right
+        view.addSubview(label)
+    }
+
+    /// создание сегмент контрала
+    private func createSegmendControll() {
+        segmentControl = UISegmentedControl(items: Constants.menuItems)
+        segmentControl.frame = .init(x: 16, y: 368, width: view.frame.width - 32, height: 44)
+        segmentControl.selectedSegmentIndex = 0
+        segmentControl.addTarget(self, action: #selector(selectedItemSegmentControl), for: .valueChanged)
+        view.addSubview(segmentControl)
+    }
+
+    /// создание кнопки
+    private func orderMethod() {
+        orderButton.center.x = view.center.x
+        orderButton.setTitle("Заказать", for: .normal)
+        orderButton.addTarget(self, action: #selector(tappedButton), for: .touchUpInside)
+        view.addSubview(orderButton)
+    }
+
+    /// Заглушки
+    private func emptyView(title: String, lineX: Int) {
+        let button = UIButton()
+        button.frame = .init(x: lineX, y: 482, width: 165, height: 165)
+        button.backgroundColor = #colorLiteral(red: 0.9686273932, green: 0.9686273932, blue: 0.9686273932, alpha: 1)
+        button.addTarget(self, action: #selector(orderDetailUp), for: .touchUpInside)
+        view.addSubview(button)
+    }
+
+    /// выбор селектора ссегмента
+    @objc private func selectedItemSegmentControl() {
+        let segmentIndex = segmentControl.selectedSegmentIndex
+        itemImage.image = imageArray[segmentIndex]
+    }
+
+    /// открываем экран с выбором дополнительных ингридиентов и кладем в нас масссив
+    @objc private func orderDetailUp() {
+        let orderVC = MenuItemIngridientsConfigurationViewController()
+        orderVC.closure = { [weak self] item in
```

Comment:
```suggestion
        orderVC.closure = { [weak self] item in
```
closure не по гайду Handler читай стайл гайд